crypto/crypto_aesctr.c

```diff
@@ -44,7 +44,7 @@ crypto_aesctr_init(const struct crypto_aes_key * key, uint64_t nonce)
 
 /**
  * crypto_aesctr_stream(stream, inbuf, outbuf, buflen):
- * Generate the next ${buflen} bytes of the AES-CTR stream and xor them with
+ * Generate the next ${buflen} bytes of the AES-CTR ${stream} and xor them with
```

Comment:
In this case, "stream" was being used as a noun rather than as a name.  Perhaps it should be "the AES-CTR stream ${stream}" and in _free "Free the AES-CTR stream ${stream}." though.